docs/ChangeLog/20191129/PR6749.md

```diff
@@ -0,0 +1,4 @@
+* Adding support for `BACKLIGHT_ON_STATE` for hardware PWM backlight
```

Comment:
Would you please move/rename this file to `docs/ChangeLog/20200229/PR6749.md`?

Do that and I'll merge this into `future` branch.